scripts/preload_models.py

```diff
@@ -17,41 +17,39 @@
 
 transformers.logging.set_verbosity_error()
 
+#---------------------------------------------
 # this will preload the Bert tokenizer fles
-print('Loading bert tokenizer (ignore deprecation errors)...', end='')
-with warnings.catch_warnings():
-    warnings.filterwarnings('ignore', category=DeprecationWarning)
-    tokenizer = BertTokenizerFast.from_pretrained('bert-base-uncased')
-print('...success')
-sys.stdout.flush()
+def download_bert():
```

Comment:
While adding the download function for the safety checker model, I felt it was important to reorganize `preload_models.py` to make it more modular (or at least understandable).

Reply:
OK, fair.

Usually one would at least put this in its own commit.